appveyor.yml

```diff
@@ -8,51 +8,34 @@ environment:
     # Avoid long paths on Windows
     STACK_ROOT: "c:\\s"
     STACK_WORK: ".w"
-    CACHE_DIR: "c:\\c"
     WORK_DIR: "c:\\w"
+    CACHE_S3_VERSION: v0.1.1
+    AWS_REGION: us-west-1
+    S3_BUCKET: appveyor-ci-cache              
+    AWS_ACCESS_KEY_ID: 
+      secure: 0OtMCUmJ4V7fNRPvKjzYxSO19I200rPG8WRW/qn1QPk=
+    AWS_SECRET_ACCESS_KEY:
+      secure: EmPAMpga8gjXxPMfJSsdq2sof8SLwyga1rrYuZDlPoriMS1iEFLvCNLA70JqHSna
```

Comment:
shouldn't these be stored in appveyor and then read as environment variables?

Reply:
They are encrypted as per https://www.appveyor.com/docs/how-to/secure-files/#decrypting-files-during-an-appveyor-build